core/src/main/scala/scalaz/std/Future.scala

```diff
@@ -5,7 +5,8 @@ import scala.concurrent.{Await, CanAwait, ExecutionContext, Future}
 import scala.concurrent.duration.Duration
 
 trait FutureInstances1 {
-  implicit def futureInstance(implicit ec: ExecutionContext): Monad[Future] with Cobind[Future] =
+
+  implicit def futureInstance(implicit ec: ExecutionContext): Cobind[Future] with MonadError[({type λ[α,β] = Future[β]})#λ, Throwable] with Catchable[Future] =
```

Comment:
I'm still a bit stuck, perhaps @xuwei-k or you can help me understand further. When I tried to *add* a method to FutureInstances I got this error

```
[error]  * method newFutureInstance(scala.concurrent.ExecutionContext)scalaz.Monad in trait scalaz.std.FutureInstances is present only in current version
[error]    filter with: ProblemFilters.exclude[ReversedMissingMethodProblem]("scalaz.std.FutureInstances.newFutureInstance")
java.lang.RuntimeException: scalaz-core: Binary compatibility check failed!
```